src/ci/bin/test.inc.sh

```diff
@@ -786,16 +785,52 @@ cromwell::private::delete_sbt_boot() {
     rm -rf ~/.sbt/boot/
 }
 
+cromwell::private::install_adoptopenjdk() {
+    # https://adoptopenjdk.net/installation.html#linux-pkg-deb
+    sudo apt-get install -y wget apt-transport-https gnupg
+    wget -qO - https://adoptopenjdk.jfrog.io/adoptopenjdk/api/gpg/key/public |
+        sudo apt-key add -
+    echo "deb https://adoptopenjdk.jfrog.io/adoptopenjdk/deb $(
+            grep UBUNTU_CODENAME /etc/os-release | cut -d = -f 2
+        ) main" |
+        sudo tee /etc/apt/sources.list.d/adoptopenjdk.list
+    sudo apt-get update
+    sudo apt-get install -y adoptopenjdk-11-hotspot
+    sudo update-java-alternatives --set adoptopenjdk-11-hotspot-amd64
+}
+
+cromwell::private::install_sbt() {
+    # https://www.scala-sbt.org/1.x/docs/Installing-sbt-on-Linux.html#Ubuntu+and+other+Debian-based+distributions
+    echo "deb https://dl.bintray.com/sbt/debian /" | sudo tee -a /etc/apt/sources.list.d/sbt.list
+    curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" |
+        sudo apt-key add
+    sudo apt-get update
+    sudo apt-get install -y sbt
+}
+
+cromwell::private::install_docker_compose() {
+    # Install or upgrade docker-compose so that we get the correct exit codes
+    # https://docs.docker.com/compose/release-notes/#1230
+    # https://docs.docker.com/compose/install/
+    curl \
+        -L "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" \
+        > docker-compose
+    sudo mv docker-compose /usr/local/bin
+    sudo chmod +x /usr/local/bin/docker-compose
+}
+
+cromwell::private::setup_pyenv_python_latest() {
+    # Make `python` whatever the most recent version of python installed
```

Comment:
I think it would be better to use explicitly `python3` and `pip3` where possible

Reply:
Explicitly done where possible.

Leaving this sync-up in to make sure Jenkins/Travis/Circle and future CIs all ensure `python` in the `$PATH` is version 3 not 2, just in case some other script/program is launching the legacy `python` or `pip` commands.